dev/pages/Modal.vue

```diff
@@ -26,7 +26,7 @@ watch(showModal, (showing) => modalShowing.value = showing)
     <component-title title="Modal" />
 
     <token>
-      <w-modal title="Hello Warp!" :style="demoStyles" :left="showLeft" :right="{ 'aria-label': 'Close' }" @dismiss="showModal = false" v-model="showModal" @right="showModal = false">
+      <w-modal title="Hello Warp!" :style="demoStyles" :left="showLeft" right @dismiss="showModal = false" v-model="showModal" @right="showModal = false">
```

Comment:
Does it mean it's not possible to override `aria-label` anymore?

Reply:
I just want to use the default aria label  (when running this project locally) to make sure that the translation work as expected 😄